Vet clinic database: create animals table

ragangithub opened the pull request:
- Created a database named vet clinic.
- Created a table named Animals with the different columns
- Inserted data into the table
- Queried data from table

![Screenshot (270)](https://github.com/ragangithub/vet-clinic-db/assets/109742772/154a863a-7a61-492c-b329-cf6db11a2f8d)
![Screenshot (269)](https://github.com/ragangithub/vet-clinic-db/assets/109742772/7bdf648b-52fe-4ef3-99a3-d08e00e98672)

ragangithub committed:
Add animals table

rloterh reviewed (commented):
Hi Ragan, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented; Ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe.

**Kindly remove/update all aspects similar to the underlined:**

![image](https://github.com/ragangithub/vet-clinic-db/assets/12745474/d084dc66-dfe9-40f8-959d-977995233961)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

ragangithub committed:
Update README

ragangithub commented:
- removed all  guide instructions

rloterh reviewed (commented):
Hi Ragan, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented; Ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe.

**Kindly remove/update all aspects similar to the underlined:**

![image](https://github.com/ragangithub/vet-clinic-db/assets/12745474/4c6f1835-235d-426d-981b-887ca68274f2)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

ragangithub committed:
Update README

ragangithub committed:
Update README

ragangithub commented:
- removed  guide instructions

Munsa1 commented on README.md:
```diff
@@ -1 +1,269 @@
+
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
```
- [ ] Kindly remove commented lines from your readme file as per requirement in line 32. This goes for lines commented through out the readme file which have no relevance

Munsa1 commented on data.sql:
```diff
@@ -0,0 +1,10 @@
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-01-07', 1, 'false', 15.04);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, 'false', 11);
```
- [ ] For `Devimon` the value for `neutered` should be set to `true` as per requirement.
![image](https://github.com/ragangithub/vet-clinic-db/assets/81932478/48b7393a-fd5a-4f82-bdc6-60e1447110aa)
check requirements [here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md)

## Important 🔴 
### Note that the screenshots have to be udated for all queries where changes are required

Munsa1 commented on queries.sql:
```diff
@@ -0,0 +1,10 @@
+
+
+SELECT * FROM animals WHERE name like '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```
- [ ] Here we are supposed to select the `name` of all animals born between 2016 and 2019 and not the entire record.
e.g
`SELECT name FROM animals`

NOT ❌ 
`SELECT * FROM animals`
- [ ] Also the date should be between `'2016-01-01' AND '2019-12-31'` and not `'2016-01-01' AND '2019-01-01'`

Munsa1 commented on queries.sql:
```diff
@@ -0,0 +1,10 @@
+
+SELECT * FROM animals WHERE name like '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT * FROM animals WHERE neutered = 'true' AND  escape_attempts < 3;
```
- [ ] Again here we are supposed to select the `name` column only not the entire record
![image](https://github.com/ragangithub/vet-clinic-db/assets/81932478/30af61dd-2507-4a5a-92f6-d5a50d6faf4e)
Check the requirements

Munsa1 commented on README.md:
```diff
@@ -1 +1,269 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
```
- [ ] Kindly remove this link from the table of contents as it points to a section which has been removed

danmainah replied:
- [ ] @ragangithub Great job implementing the changes requested, however you forgot to remove the links to the FAQ section that is not there in your readme file

Munsa1 commented on README.md:
```diff
@@ -1 +1,269 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- <details>
+  <!-- <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details> -->
+
+<!-- <details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details> -->
```
- [ ] Again here kindly remove commented lines for a cleaner more professional readme

Munsa1 commented on README.md:
```diff
@@ -1 +1,269 @@
+- **[Queries to get back data from the table]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](<replace-with-your-deployment-URL>)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
```
- [ ] Again here kindly remove commented lines for a cleaner more professional readme

Munsa1 commented on README.md:
````diff
@@ -1 +1,269 @@
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
````
- [ ] Again here kindly remove commented lines for a cleaner more professional readme

Munsa1 commented on README.md:
````diff
@@ -1 +1,269 @@
+```
+--->
+
+<!-- ### Usage
+
+To run the project, execute the following command: -->
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+<!-- ### Run tests
+
+To run tests, run the following command: -->
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+<!-- ### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
````
- [ ] Again here kindly remove commented lines for a cleaner more professional readme

Munsa1 commented on README.md:
```diff
@@ -1 +1,269 @@
+- GitHub: [Ragan](https://github.com/ragangithub)
+- LinkedIn: [Ragan Hailu](https://linkedin.com/in/raganhailu)
+
+<!-- 👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle) -->
```
- [ ] Again here kindly remove commented lines for a cleaner more professional readme

Munsa1 commented on README.md:
```diff
@@ -1 +1,269 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+<!-- ## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
```
- [ ] Again here kindly remove commented lines for a cleaner more professional readme

Munsa1 commented on README.md:
```diff
@@ -1 +1,269 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<!-- ## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
```
- [ ] Again here kindly remove commented lines for a cleaner more professional readme

Munsa1 reviewed (changes requested):
Hi @ragangithub, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Clean Codebase :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Followed Given File Structure :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ragangithub committed:
Update README and SQL

ragangithub commented:
- Removed comments from README
- Changed   Devimon's neutered value to true
- Changed date value from 2019-12-31' to'2019-01-01'
- Displayed the names only as suggested by the reviewer
![Screenshot (270)](https://github.com/ragangithub/vet-clinic-db/assets/109742772/6fabe5d4-16fc-4775-a1f3-6a48b1272a2c)

![Screenshot (269)](https://github.com/ragangithub/vet-clinic-db/assets/109742772/a30a02bd-9c7c-459e-b34b-8beddd3da2fc)

danmainah reviewed (changes requested):
Hi @ragangithub , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️  Implemented changes requested to ensure the use of the right data and correct queries.
✔️ Professional written README file with all sections required.


 # Required Changes ♻️

- [ ] @ragangithub Great work showing the updated screenshots of the changes requested however I would recommend you update the screenshots queries in the pull request summary as previously requested.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ragangithub committed:
Update README

ragangithub commented:
- removed FAQ section
- updated screenshot for queries
![Screenshot (270)](https://github.com/ragangithub/vet-clinic-db/assets/109742772/154a863a-7a61-492c-b329-cf6db11a2f8d)
![Screenshot (269)](https://github.com/ragangithub/vet-clinic-db/assets/109742772/7bdf648b-52fe-4ef3-99a3-d08e00e98672)

OyePriscilla reviewed (approved):
Hi @ragangithub 👍, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![200](https://user-images.githubusercontent.com/103386183/196424328-985f6e98-f7c6-4cf9-bafe-158436f70560.gif)

## Highlights
Added query's result to the PR ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
